### ISSUE TITLE
Are there any plans to upgrade Zookeeper and Kafka?

### ISSUE THREAD

**Issue by instinct-vfx** [getsentry/self-hosted#1563]
### Version

22.6.0

### Steps to Reproduce

As of 22.6.0 Sentry still uses versions 5.5.0 of both Zookeeper and Kafka. This causes security scans to flag our sentry deployment. I am aware that Confluent is maintaining a private fork of log4j 1.x that addresses CVEs, however it is really hard to follow up individual CVEs and proove to our security teams that CVEs are addressed.

Are there any plans to update to current versions of Confluent's releases of Kafka and Zookeeper?

### Expected Result

Current versions of Zookeeper and Kafka being used

### Actual Result

Zookeeper and Kafka are still 5.5.0

**Comment by ethanhs** [comment]
We definitely should upgrade, I think it is a matter of bumping the versions and making sure nothing breaks. It looks like they don't really keep a changelog that I can see...

**Comment by ethanhs** [comment]
Thanks! That is useful. I'll try updating things a bit later and see how it goes.

**Comment by aminvakil** [comment]
We're tracking outdated java jre/jdk of kafka/zookeeper in https://github.com/getsentry/self-hosted/issues/1292, @ethanhs what do you think about continuing this discussion there too?

**Comment by ethanhs** [comment]
Yeah that seems reasonable, I will close in favor of that issue.
